libraries/cmake/formula/openssl/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 3.14.5)
+project(thirdparty_openssl)
+
+include(ExternalProject)
+
+function(opensslMain)
+  generateMetadataTargets()
+
+  if(NOT DEFINED thirdparty_zlib_INCLUDE_DIRS OR
+     NOT DEFINED thirdparty_zlib_LIBRARIES)
+
+    message(WARNING "The zlib dependency was not specified correctly")
+    return()
+  endif()
+
+  # The formula will be configured with the C_FLAGS and CXX_FLAGS parameters; we should
+  # forward them to the configure script when possible
+  message(WARNING "TODO: Use C_FLAGS")
```

Comment:
We are not yet passing the compiler/linker flags to the openssl configure script. We receive them as C_FLAGS and CXX_FLAGS from the main osquery project